css/properties/text-decoration-skip.json

```
@@ -41,23 +41,23 @@
             "safari": [
               {
                 "version_added": "12.1",
-                "notes": "Only supports the <code>none</code> and <code>skip</code> values; all other values behave like those two values."
+                "notes": "Only supports the <code>none</code> and <code>auto</code> values; all other values behave like those two values."
```

OK, this was a bit of a time waster, but I wondered what this note actually meant. All other values behave _like_ those two values? What?

Turns out, this data is pretty rough. It's from the old wiki tables and it shows. I tested this and found that an additional value is supported and the version numbers are wrong. I think the note ought to read:

```suggestion
                "notes": "Supports only <code>none</code>, <code>auto</code>, and <code>objects</code> values."
```

Also the versions supported are wrong for the prefixed version. The prefixed version appears to have shipped in 6.1 or 6.2, so based on the discussion in https://github.com/mdn/browser-compat-data/issues/9423, I'd set that to 7.